packages/components/charts/src/pie-chart-label.tsx

```diff
@@ -22,6 +22,7 @@ export interface PieChartLabelProps {
   percent?: number
   value?: number
   labelProps?: HTMLUIProps<"text">
+  percentProps?: boolean
```

Comment:
change `percentProps` to `calcPercent`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3609/files?w=1#diff-9a87e202208abe15a402535a5df24da558365facd146111b8202b00c0e28f218R30